docs/localization.md

```diff
@@ -25,9 +25,9 @@ The structure of the locale file is typically like so:
   epiphany: {},
   ordinaryTime: {},
```

Comment:
@emagnier, why `holy_week` and `liturgical_colors` below are in snake_case, but `ordinaryTime` not?

Reply:
Because I forgot to change it to snake_case. Good catch!